examples/dynamic-ubuntu-wait/packer-build.json

```
@@ -0,0 +1,47 @@
+{
+  "min_packer_version": "0.12.0",
+  "variables": {
+    "aws_region": "us-west-1",
+    "instance_type": "",
+    "module_branch": ""
+  },
+  "builders": [{
+    "name": "dynamic-wait-test-ami-ubuntu",
+    "ami_name": "gruntwork-ubuntu-dynamic-wait-example-{{uuid | clean_resource_name}}",
+    "ami_description": "An Ubuntu AMI that has Elasticsearch installed.",
+    "instance_type": "{{user `instance_type`}}",
+    "region": "{{user `aws_region`}}",
+    "type": "amazon-ebs",
+    "source_ami_filter": {
+      "filters": {
+        "virtualization-type": "hvm",
+        "architecture": "x86_64",
+        "name": "ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-*",
+        "block-device-mapping.volume-type": "gp2",
+        "root-device-type": "ebs"
+      },
+      "owners": ["099720109477"],
+      "most_recent": true
+    },
+    "ssh_username": "ubuntu"
+  }],
+  "provisioners": [{
+    "type": "shell",
+    "inline": [
+      "# Get around issue where automatic ubuntu updates prevent package installation.",
+      "curl -Ls https://raw.githubusercontent.com/gruntwork-io/bash-commons/{{user `module_branch`}}/modules/bash-commons/src/dynamic-ubuntu-wait.sh | bash",
+      "sudo apt-get -y update"
+    ]
+  },{
+    "type": "shell",
+    "inline": [
+      "sudo apt-get install -y rolldice"
+    ]
+  },{
+    "type": "shell",
+    "inline": [
+      "sudo apt-get -y update",
+      "sudo apt-get install -y nano"
```

```suggestion
      "sudo DEBIAN_FRONTEND=noninteractiveapt-get -y update",
      "sudo DEBIAN_FRONTEND=noninteractiveapt-get install -y nano"
```

Why are we calling `update` again here, BTW?

---

Just realized this is a redundant test check with the rolldice step so removed.